frontend/src/tests/import-export-data/applicants_missing_required_keys.csv

```diff
@@ -0,0 +1,5 @@
+﻿Last Name,First Name,UTORid,Student Number,email,Phone, age
+Doe,John,,10000000,goofy-duck@donald.com,4166666666,
```

Comment:
Why are the numbers quoted in `applicants_correct.csv` and not in this one?

Reply:
sorry... forgot to change this one after fixing the applicants_correct.csv